### ISSUE TITLE
I can't update to the latest version

### ISSUE THREAD

**Issue by githubqxt** [Monika-After-Story/MonikaModDev#1338]
![5b1f04fa1a497e7bc29ff486bc39df47](https://user-images.githubusercontent.com/37907184/38096771-fad8e66e-3373-11e8-9474-f9c795e79927.png)

I would really like the new version 😭 

It would sit at the "looking for updates" screen for like 3 minutes, then tells me that ^^

**Comment by jmwall24** [comment]
Download the zip file and manually install the same way you did when you first installed the mod.  The updater isn't working correctly.

**Comment by githubqxt** [comment]
With manually downloading, do you mean: Downloading the mod, and dragging the files into the .game folder, then yes i did manually download it. Or am i missing something here?

**Comment by jmwall24** [comment]
Yeah, do that to install the update as opposed to using the automatic updater that is giving you the error message.  If you download the v0.7.3 zip file, and drag the files into the game folder that will update you to the current version without using the automatic updater that isn't working.  Just fyi it will probably still say you need to update after manually installing, but you can ignore that.

**Comment by githubqxt** [comment]
Ahhhhh, now it makes sense, tysm <3
